### PR TITLE
added preventCache back in

### PR DESCRIPTION
**Pull request by bob-bins**

### CODE REVIEW

**Comment by bob-bins** [comment]
passing `onMasterMerge` test: https://circleci.com/gh/heartpoints/heartpoints.org/189

File: Dockerfile [hunk 1]
```diff
@@ -1,8 +1,16 @@
 FROM node:10.14.2
  
 WORKDIR /heartpoints.org
-COPY . .
-RUN ./heartpoints.sh prepareForRun
+
+COPY package.json yarn.lock ./
+RUN yarn install
```
Comment (tommysullivan) [inline Dockerfile]: Yassssss!

File: Dockerfile [hunk 2]
```diff
@@ -1,8 +1,16 @@
...
+
+COPY src src
+COPY tsconfig.json webpack.config.js ./
+RUN yarn webpack --verbose
+
+COPY heartpoints.sh ./
```
Comment (tommysullivan) [inline Dockerfile]: Yaaaaasssssssss!!!!!

File: .circleci/config.yml [hunk 3]
```diff
@@ -10,7 +10,7 @@ jobs:
       - run: ./heartpoints.sh onPullRequest
   onMasterMerge:
     docker:
-      - image: gcr.io/heartpoints-org/cicd:1.0.0
+      - image: gcr.io/heartpoints-org/cicd:1.0.1
```
Comment (bob-bins) [inline .circleci/config.yml]: this process is clunky

File: Dockerfile [hunk 4]
```diff
@@ -1,8 +1,16 @@
 FROM node:10.14.2
  
 WORKDIR /heartpoints.org
-COPY . .
```
Comment (aashreya) [inline Dockerfile]: @bob-bins What's the reason for this not working ?

**Pull request merged**